File: Round2/Find Minimum in Rotated Sorted Array/code.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int nums[] = new int[n];
        for(int i=0; i<n; i++) {
            nums[i] = sc.nextInt();
        }
        int min = Integer.MAX_VALUE;
        for(int x : nums) {
            min = Math.min(min,x);
        }
        System.out.println(min);
    }
}

File: Round2/Find Minimum in Rotated Sorted Array/code.cpp
#include <iostream>
#include <limits.h>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    int nums[n];
    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }
    
    int min = INT_MAX;
    for (int i = 0; i < n; i++) {
        if (nums[i] < min) {
            min = nums[i];
        }
    }
    
    cout << min << endl;
    return 0;
}
